CHANGELOG

```diff
@@ -9,6 +9,7 @@ v3.15 (* 2019)
  - Limit project name length when viewing a project
  - Removed bullet style in node modals.
  - Fix overflow issue where content would expand out of view.
+ - Set the "by" attribute when using the API
```

Comment:
maybe specify this is for paper-trail, or even better for internal versioning / trash feature? otherwise the line doesn't give much to a user reading the CHANGELOG.